Alignment/HIPAlignmentAlgorithm/plots/HIPplots.cc

```diff
@@ -0,0 +1,1761 @@
+      if(etapeakcont>=etapeakmax){
+	etapeakmax=etapeakcont;
+	highestetapeak=bin;
+      }
+    }
+    //  float etapeakwidth=(etabins[highestetapeak+1]-etabins[highestetapeak]);
+    float etalim1,etalim2;
+    if(subDet == TIBid){
+        //      etalim1 =(etapeaks[highestetapeak]-etabins[highestetapeak-1]);
+      etalim1 =(etabins[highestetapeak-1])+0.05;
+      etalim2=(etapeaks[highestetapeak+1]+etapeaks[highestetapeak])/2.0;
+      }
+    else{
+        etalim1 =(etapeaks[highestetapeak-1]+etapeaks[highestetapeak])/2.0;
+      etalim2=(etapeaks[highestetapeak+1]+etapeaks[highestetapeak])/2.0;
+    }
+
+    //    float etalim1=etabins[highestetapeak]-etapeakwidth/4.0;
+    //    float etalim2=etabins[highestetapeak]+etapeakwidth/4.0;
+    if(etalim1<0.0||etalim2<0.0){//oooops! we run out of range..
+      if(subDet == TIBid){
+	etalim1 =(etabins[highestetapeak-2])+0.05;
+	etalim2=(etapeaks[highestetapeak]+etapeaks[highestetapeak-1])/2.0;
+      }
+      else{
+	etalim1=(etapeaks[highestetapeak-2]+etapeaks[highestetapeak-1])/2.0;
+	etalim2=(etapeaks[highestetapeak]+etapeaks[highestetapeak-1])/2.0;
+      }
+    }
+
+    char etasel_str[64];
+    sprintf(etasel_str,"Eta>=%f&&Eta<%f",etalim1,etalim2);
+    TCut seleta=etasel_str;
+    int nphietapeak=talignable->Draw("Phi",commonsense&&selA&&seleta,"goff");
+    
+    const int Nphipoints= nphietapeak;
+    float phipoints[Nphipoints];
+    cout<<"Found "<<Nphipoints<<" phi points in "<<etasel_str<<endl;
+    //    Nphipoints= Nphipoints*2;
+    for(bin=0;bin<Nphipoints;bin++){
+      phipoints[bin]=talignable->GetV1()[bin];
+    }
+    //    stable_sort(phipoints.begin(),phipoints.end());
+    stable_sort(phipoints,phipoints+Nphipoints);
+
+//     for(bin=0;bin<Nphipoints;bin++){
+//       cout<<"\t"<<phipoints[bin]<<flush;
+//     }
+//     cout<<"---- End Phi points"<<endl<<endl;
+
+
+    const int Nphibins=Nphipoints+1;
+    float phibins[Nphibins];
+
+    for(int phibin=0;phibin<Nphipoints;phibin++){
+	//	float phidiff=0.0;
+	if(phibin%20==1)cout<<"Phi Bin #"<<phibin<<"  "<<flush;
+	if(phibin==0){
+	  float phidiff=(phipoints[phibin+1]- phipoints[phibin]);
+	  phibins[phibin]=phipoints[phibin]-phidiff/2.0;
+	}
+	
+	else{
+	  float res=(phipoints[phibin]+phipoints[phibin-1])/2.0;
+	  //cout<<"Phipoints: "<<phipoints[phibin]<<"\tRES= "<<res<<endl;
+	  phibins[phibin]=res;
+	  //	  cout<<phibins_tmp[bin][phibin]<<endl;
+	  if(phibin==Nphipoints-1){
+	  float phidiff=(phipoints[phibin]- phipoints[phibin-1]);
+	  phibins[phibin+1]=phipoints[phibin]+phidiff/2.0;
+	}
+	}
+	 
+	if(phibin%20==1)cout<<phibins[phibin]<<endl;
+    }//end loop on phi bins
+
+   
+   // cout<<"Filled phi binning: "<<endl;
+    //for(bin=0;bin<Nphibins;bin++){
+    //  cout<<"\t"<<phibins[bin]<<flush;
+   // }
+   // cout<<endl;//<<endl;
+   
+    ////////////
+    cout<<"Finished to fill final phi binning"<<endl;
+    //-------(1) END BINNING part ---
+
+*/
```

Comment:
@CandiceYou this file contains quite a few commented lines, especially this last part. If you don't use this code anymore, you should remove it to ease maintenance for future developers.